lib/bigtable/row.js

```diff
@@ -0,0 +1,285 @@
+ * });
+ *
+ * var table = bigtable.table('my-table');
+ * var row = table.row('lincoln');
+ */
+function Row(table, name) {
+  var methods = {
+
+    /**
+     * @param {function} callback - The callback function
+     * @param {?error} callback.err - Request error
+     * @param {boolean} callback.exists
+     *
+     * @example
+     * row.exists(function(err, exists) {});
+     */
+    exists: true
+  };
+
+  var config = {
+    parent: table,
+    methods: methods,
+    id: name
+  };
+
+  GrpcServiceObject.call(this, config);
+}
+
+nodeutil.inherits(Row, GrpcServiceObject);
+
+/**
+ * @param {object} entry - Row entry. See {module:bigtable/table#insert}
+ * @param {function} callback - The callback function
+ * @param {?error} callback.error - Request error
+ * @param {object} callback.apiResponse - Request response
+ *
+ * @example
+ * var entry = {
+ *   name: {
+ *     first: 'Abraham',
+ *     last: 'Lincoln'
+ *   }
+ * };
+ *
+ * row.create(entry, function(err, apiResponse) {});
+ */
+Row.prototype.create = function(entry, callback) {
+  this.parent.insert({
+    key: this.id,
+    data: entry
+  }, callback);
+};
+
+/*!
+ * @TODO should the deleteCellsFrom version be moved to a different method
+ * like "remove" or "deleteCellsFrom"?
+ */
+/**
+ * @param {=string[]} qualifiers - Qualifiers for cells to be deleted.
+ *     See {module:bigtable/table#deleteCells}
+ * @param {function} callback - The callback function
+ * @param {?error} callback.err - Request error
+ * @param {object} callback.apiResponse - Full response
+ *
+ * @example
+ * //-
+ * // You can pass in an array of qualifiers to be deleted
+ * //-
+ * row.delete(['name:first'], function(err, apiResponse) {});
+ *
+ * //-
+ * // Or just delete all the row cells by omitting qualifiers
+ * //-
+ * row.delete(function(err, apiResponse) {});
+ */
+Row.prototype.delete = function(qualifiers, callback) {
+  var self = this;
+
+  if (is.function(qualifiers)) {
+    callback = qualifiers;
+    qualifiers = [];
+  }
+
+  var entries = qualifiers.map(function(qualifier) {
+    return {
+      key: self.id,
+      data: Entry.parseQualifier(qualifier)
+    };
+  });
+
+  if (!entries.length) {
+    entries.push({ key: self.id });
+  }
+
+  this.parent.deleteCellsFrom(entries, callback);
+};
+
+/*!
+ * @TODO consider moving this to something like .getCells() ?
+ */
+/**
+ * @param {=string[]} qualifiers - List of qualifiers to retrieve
+ * @param {function} callback - The callback function
+ * @param {?error} callback.err - Request error
+ * @param {module:bigtable/row} callback.row - Row instance
+ * @param {object} callback.apiResponse - Request response
+ *
+ * @example
+ * //-
+ * // Use this method to grab an entire row
+ * //-
+ * row.get(function(err, row, apiResponse) {});
+ *
+ * //-
+ * // Or pass in an array of qualifiers to only retrieve certain cells
+ * //-
+ * row.get(['name:first'], function(err, row, apiResponse) {});
```

Comment:
@stephenplusplus I combined these together, do you think fetching individual cells would be better moved to `row.getCells(['name:first'])` ..?